packages/office-addin-manifest/src/manifestHandler/manifestHandler.ts

```diff
@@ -2,27 +2,12 @@
 // Licensed under the MIT license.
 
 import { ManifestInfo } from "../manifestInfo";
-import * as util from "util";
-import * as fs from "fs";
 
 export abstract class ManifestHandler {
   /* eslint-disable no-unused-vars */
-  constructor() {
-    this.manifestPath = "";
-    this.fileData = "";
-  }
-
-  public async readFromManifestFile(manifestPath: string): Promise<string> {
+  constructor(manifestPath: string) {
     this.manifestPath = manifestPath;
-    try {
-      const fileData: string = await util.promisify(fs.readFile)(manifestPath, {
-        encoding: "utf8",
-      });
-      this.fileData = fileData;
-      return fileData;
-    } catch (err) {
-      throw new Error(`Unable to read data for manifest file: ${manifestPath}. \n${err}`);
-    }
+    this.fileData = "";
```

Comment:
Why do we need this intermediate object?  It's never read and only stored in one method (which just returns the same data anyway.

Unless you change the method that accepts or return the data as arguments to just use the internal property instead . . . but I'm not sure if that would be a breaking change from how things work right now (exported API methods).

Reply:
It is only currently needed for the `xml` case. I will move the manipulation of this variable to the `xml` class.